logging/nox.py

```diff
@@ -23,6 +23,14 @@
     os.path.join('..', 'api_core'),
     os.path.join('..', 'core'),
 )
+UNIT_TEST_DEPS = (
+    'mock',
+    'pytest',
+    'pytest-cov',
+    'flask',
+    'webapp2',
+    'webob',
+)
```

Comment:
Ugh. FWIW, these can hardly be considered "unit" test dependenies.  Ideally, the code which provides integration for the various web frameworks wouldn't even be packaged together with `google.cloud.logging`.

Reply:
@tseaver I agree, though it's "optional" (but not listed as an extra)